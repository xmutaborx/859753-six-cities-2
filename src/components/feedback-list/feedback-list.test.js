import React from 'react';
import renderer from 'react-test-renderer';
import FeedbackList from './feedback-list.jsx';

import COMMENT_MOCK from '../../mocks/comment-mock';

it(`Feedback correctly renders`, () => {
  const tree = renderer
    .create(<FeedbackList
      comments={COMMENT_MOCK}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
