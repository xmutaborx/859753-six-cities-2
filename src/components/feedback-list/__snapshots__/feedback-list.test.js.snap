// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback correctly renders 1`] = `
Array [
  <h2
    className="reviews__title"
  >
    Reviews Â· 
    <span
      className="reviews__amount"
    >
      1
    </span>
  </h2>,
  <ul
    className="reviews__list"
  >
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height="54"
            src="src"
            width="54"
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Name
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "100%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          test comment
        </p>
        <time
          className="reviews__time"
          dateTime="2019-04-24"
        >
          08.12.2019
        </time>
      </div>
    </li>
  </ul>,
]
`;
